api/compute/v1alpha1/function_webhook.go

```diff
@@ -194,6 +194,23 @@ func (r *Function) ValidateCreate() error {
 		allErrs = append(allErrs, fieldErrs...)
 	}
 
+	if r.Spec.Pod.VPA != nil {
+		if r.Spec.MaxReplicas != nil {
+			allErrs = append(allErrs,
+				field.Invalid(field.NewPath("spec").Child("pod").Child("vpa"), r.Spec.Pod.VPA, "you can not enable HPA and VPA at the same time"))
+		}
+
+		if r.Spec.Pod.VPA.ResourcePolicy != nil {
```

Comment:
Seems we can create a new function for this check.

Reply:
I will update